config/settings/test.yml

```diff
@@ -411,3 +411,6 @@ ask_va:
 
 nod_vanotify_status_callback:
   bearer_token: bearer_token_secret
+
+pension_ipf_vanotify_status_callback:
```

Comment:
Has this been added to the live envs yet?

Reply:
@LindseySaari No this has not. I don't have write access to the `vsp-infra-application-manifests` repo, but i have a diff of the changes needed.

Reply:
If you are in the gov-team-write github team, you should have access to add these

Reply:
Yup, these have since been added to staging.